server/src/test/java/org/opensearch/monitor/os/OsProbeTests.java

```diff
@@ -295,6 +296,7 @@ List<String> readSysFsCgroupCpuAcctCpuStat(String controlGroup) throws IOExcepti
             }
         };
 
+        assumeThat("CGroups are not available", noCpuStatsOsProbe.areCgroupStatsAvailable(), is(true));
```

Comment:
Can we add a comment here and mention the reason to assume is because docker faces error while reading control group stats?

Reply:
For my education, since we have https://github.com/opensearch-project/OpenSearch/pull/2101 merged on main already, why did we encounter https://github.com/opensearch-project/OpenSearch/issues/2592?

Reply:
> Can we add a comment here and mention the reason to assume is because docker faces error while reading control group stats?

@owaiskazi19 I have not been able to reproduce https://github.com/opensearch-project/OpenSearch/issues/2592 exactly, but the original comment https://github.com/opensearch-project/OpenSearch/pull/2067#issuecomment-1135160622 is related to the fact that CGroups stats is not available (this backport).

Reply:
Did you try pulling [opensearchstaging/ci-runner:ci-runner-ubuntu1804-build-v1](https://hub.docker.com/layers/opensearchstaging/ci-runner/ci-runner-ubuntu1804-build-v1/images/sha256-2c7bb2780bc08cd4e7e3c382ac53db414754dabd52f9b70e1c7e344dfb9a0e5e?context=explore) and run gradle check on the same? This is a new docker image actually for public jenkins work.

Reply:
I will, I used official 18.04 which also failed (not anymore with the this fix)